app.json

```diff
@@ -255,7 +315,31 @@
         ]
       },
       {
-        "id": "wind_below",
+        "id": "wind_above_tempest",
```

Comment:
Do we really need unique triggers for tempest? Can we use the same and filter on what kind of device as a setting instead?

Reply:
Makes sense to use the same trigger but with a device filter. I can fix it within the next couple of days if you want. Or will you fix it?

Reply:
I think you should fix it and test it with tempest, then I can verify that it still works with sky

Reply:
I made a quick test where one common trigger, with two args of type device, is used. But it did not work as expected. I get the "Error: invalid_flow_card_id". I now remember trying that earlier on as well before going for separate ones. Can you think of something, maybe in the app/onInit, that could cause this behavior?

Reply:
Take a look at, https://github.com/rogro82/com.hue.zigbee/blob/master/app.json, where device is an argument to the trigger.

Reply:
Did you understand from this sample how it can be done?

Reply:
Have you given up? 😀

Reply:
I have fixed it now.

Reply:
Sounds great. Will install tonight and try it out, just need some more light to power the Tempest :)